paasta_tools/kubernetes_tools.py

```diff
@@ -1964,6 +1969,15 @@ def list_deployments(
     ]
 
 
+def list_kubernetes_services(kube_client: KubeClient,) -> Sequence[KubernetesService]:
+    services = kube_client.core.list_namespaced_service(namespace="paasta")
+
+    return [
+        KubernetesService(name=item.metadata.name, labels=item.metadata.labels)
+        for item in services.items
+    ]
+
+
```

Comment:
this and `KubernetesService` don't seem like they have enough meat to be separate entities, let's just use kubernetes entities directly